recipes/partsegdata/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "partsegdata" %}
+{% set version = "0.10.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/PartSegData-{{ version }}.tar.gz
+  sha256: 1a48d0f8eb394420879a6308f2917f13ca9e525c4dc16d1f7d870b6fd5393d64
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+
+test:
+  imports:
+    - PartSegData
+  commands:
+    - pip list
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://4dnucleome.cent.uw.edu.pl/PartSeg/
+  summary: PartSeg data files
+  license: BSD-3-Clause
+  license_file: License.txt
+
+extra:
+  recipe-maintainers:
+    - conda-forge/napari
```

Comment:
This one does not depend on napari, but I assume is a dependency of a plugin. Do we still want the team to be maintaining this one too?

Reply:
Ok, fine, I'll leave it if it's a split then.

Reply:
Hm, come to think about it, if they are so intimately related, maybe a multi-output recipe is better suited... Would these three packages be updated at the same time or do they follow different release strategies @czaki?

Reply:
No. PartSeg is updated much more often.  PartSegData PartSegCore-compiled-backend are updated rarely, only if needed.

Reply:
Then separate feedstocks it is. Thanks!